File: iniciante/python/1048-aumento-de-salario.py
def aumento_salario(salario):
    if 0.0 <= salario <= 400.0:
        porcentagem = 0.15
        reajuste_ganho = porcentagem * salario
        novo_salario = salario + reajuste_ganho
    elif 400.01 <= salario <= 800.0:
        porcentagem = 0.12
        reajuste_ganho = porcentagem * salario
        novo_salario = salario + reajuste_ganho
    elif 800.01 <= salario <= 1200.0:
        porcentagem = 0.10
        reajuste_ganho = porcentagem * salario
        novo_salario = salario + reajuste_ganho
    elif 1200.01 <= salario <= 2000.0:
        porcentagem = 0.07
        reajuste_ganho = porcentagem * salario
        novo_salario = salario + reajuste_ganho
    elif salario > 2000.0:
        porcentagem = 0.04
        reajuste_ganho = porcentagem * salario
        novo_salario = salario + reajuste_ganho
    return print(f'Novo salario: {novo_salario:.2f}\n'
                 f'Reajuste ganho: {reajuste_ganho:.2f}\n'
                 f'Em percentual: {int(porcentagem * 100)} %')


valor_salario = float(input())
aumento_salario(valor_salario)

File: iniciante/c/1048-aumento-de-salario.c
#include <stdio.h>

int main(){
    float salario, novo_salario, reajuste;
    int percentual;

    scanf("%f", &salario);
    
    if (salario >= 0.0 && salario <= 400.0){
        novo_salario = salario * 1.15;
        reajuste = novo_salario - salario;
        percentual = 15;
    }
    else if (salario >= 400.01 && salario <= 800.0){
        novo_salario = salario * 1.12;
        reajuste = novo_salario - salario;
        percentual = 12;
    }
    else if (salario >= 800.01 && salario <= 1200.0){
        novo_salario = salario * 1.1;
        reajuste = novo_salario - salario;
        percentual = 10;
    }
    else if (salario >= 1200.01 && salario <= 2000.0){
        novo_salario = salario * 1.07;
        reajuste = novo_salario - salario;
        percentual = 7;
    }
    else if (salario >= 2000.0){
        novo_salario = salario * 1.04;
        reajuste = novo_salario - salario;
        percentual = 4;
    }

    printf("Novo salario: %.2f\n", novo_salario);
    printf("Reajuste ganho: %.2f\n", reajuste);
    printf("Em percentual: %d %%\n", percentual);

    return 0;
}
